core/queries/GetArticleEditable.js

```diff
@@ -0,0 +1,17 @@
+import gql from "graphql-tag";
+
+export default gql`
+  query getArticle($id: ID!) {
```

Comment:
Naming diff between file and operation name. Plus from the query I can't see where the "Editable" constraint is placed.